assets/js/src/core/modules/asset/editor/types/folder/tab-manager/tabs/preview/flex-container.tsx

```diff
@@ -51,6 +54,14 @@ const FlexContainer = (props: FlexContainerProps): React.JSX.Element => {
       })
     }
 
+    const clearCache = (): void => {
```

Comment:
Shouldn't we do this somewhere inside of the action so that the cache clears in all cases when the asset gets deleted? And in addition to that shouldn't we use our new `useElementRefresh` then?

Reply:
We cant use `useElementRefresh` because the hook cant be called without the `id` and we only get this inside the loop or did i miss something?

Reply:
I agree on adding it to the `use-delete` hook. Thers no case where we don't need to clear the cache - i guess

Reply:
improved behaviour with https://github.com/pimcore/studio-ui-bundle/pull/884/commits/e240521c46a8590c89adaa3f7be7f7ac843d5461